### PR TITLE
Adding the layout structure files

### PR DESCRIPTION
**Pull request by josepplloo**
## Summary

- Data for page content added.
- Layout styles added.

## Screenshots

image

## Trello Card

https://trello.com/c/yeqfB3Kg/23-content-layout

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -1,25 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (nmartinelli23) [inline maquetado/index.html]: Remove extra line, please.

File: maquetado/index.html [hunk 2]
```diff
@@ -1,25 +1,125 @@
...
 <head>
   <meta charset="UTF-8">
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
   <title>HTML & CSS Training</title>
 </head>
+
```
Comment (nmartinelli23) [inline maquetado/index.html]: Remove extra line, please.

File: maquetado/index.html [hunk 3]
```diff
@@ -1,25 +1,125 @@
...
+      <h3 class="tech-name">ReactJS</h3>
+      <h6 class="tech-release">
+        Release: 2013
+      </h6>
+      <p class="tech-description">
+        React is a declarative component based JavaScript library for
+        building user interfaces.
+      </p>
+    </div>
+
```
Comment (nmartinelli23) [inline maquetado/index.html]: Remove extra line, please.

File: maquetado/scss/layout.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.techs {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 48px 0 0 152px;
```
Comment (henryzarza) [inline maquetado/scss/layout.scss]: I recommend you change this for: `max-width: 100%; width: 1000px` (`1000px` is a example you can replace this for another value)

File: maquetado/scss/layout.scss [hunk 5]
```diff
@@ -1,5 +1,7 @@
 .techs {
   display: flex;
   flex-wrap: wrap;
-  padding: 48px 0 0 152px;
+  width: 1024px;
+  max-width: 100%;
```
Comment (henryzarza) [inline maquetado/scss/layout.scss]: Remember order by alphabetically

File: maquetado/scss/layout.scss [hunk 6]
```diff
@@ -1,5 +1,7 @@
...
+  margin: 70px auto 0;
```
Comment (henryzarza) [inline maquetado/scss/layout.scss]: 👆

**Pull request merged**